src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,40 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import qna.CannotDeleteException;
+
+@Embeddable
+@Access(AccessType.FIELD)
+public class Answers {
+
+    @OneToMany(mappedBy = "question")
+    private List<Answer> values = new ArrayList<>();
+
+    public Answers() {
+    }
+
+    public Answers(List<Answer> values) {
+        this.values = values;
```

Comment:
```suggestion
        this.values.addAll(values);
```

JPA를 사용할 경우 Collection의 값을 변경하는 것은 주의해야 합니다. 어떠한 이유 때문인지 JPA Proxy에 대해 학습해보아요.

Reply:
해당 코드가 필요없어져서 삭제를 하긴 했는데요 혹시 말씀해주신 키워드에 대해 조금 더 구체적으로 알려주실 수 있을까요? 책을 찾아봤을 때 엔티티에 컬렉션이 있으면 컬렉션을 관리할 목적으로 내장 컬렉션으로 변경한다는 것을 읽었습니다. 컬렉션을 변경하면 이 내장 컬렉션이 변경되어야 해서 비용이 많이 드는 것일까요?

Reply:
JPA를 통해 가져온 List는 일반적인 List가 아닌 Proxy로 감싸져서 동작하게 되는데요.
EntityManager가 해당 Proxy를 통해 여러가지 기능을 제공하다보니 List를 재선언하게 되면 문제가 발생할 수 있는 부분이 많이 있습니다 :)